### ISSUE TITLE
Remove "Scan for old IH 0.x and 1.0 'original' images"

### ISSUE THREAD

**Issue by lat9** [DivaVocals/zen_Image-Handler#79]
Going down a plugin-simplification route ... does the subject functionality need to be carried forward?

**Comment by Zen4All** [comment]
Nope, IMO people users such old software should manually delete the images.

**Comment by lat9** [comment]
Thanks for the feedback, @Zen4All ... I'll wait a bit to see if anyone else chimes in!

**Comment by DivaVocals** [comment]
Honestly, I nver felt this feature was necessary. There were NO SEO issues for any of my clients when I moved them to the new image naming convention from the original MD Hash image names. 


I simply wiped out the ENTIRE image cache and started over. The new image naming convention made image management easier, allowed us to provide links to actual images because they were named based on the original image name (and I encouraged clients to name their images after their products), and it improved SEO for images.
